pom.xml

```diff
@@ -51,7 +51,7 @@
         <apache.httpclient.version>4.5.3</apache.httpclient.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
         <jersey.version>2.28</jersey.version>
-        <jetty.version>9.4.20.v20190813</jetty.version>
+        <jetty.version>9.4.23.v20191118</jetty.version>
```

Comment:
Curious if the version Bump of Jetty was just a version bump for freshness sake, or did this bump unlock OpenID support.

Would be good to comment that in the commit message, so that if the version bump breaks other things downstream, they can know why it was bumped.

Reply:
It's the first version that supports OIDC. Will add it to the commit message good idea